presto-password-authenticators/src/main/java/io/prestosql/plugin/password/ldap/LdapAuthenticator.java

```diff
@@ -213,9 +213,8 @@ private String validateGroupMembership(String user, DirContext context)
             String userDistinguishedName = search.next().getNameInNamespace();
             while (search.hasMore()) {
                 if (!userDistinguishedName.equals(search.next().getNameInNamespace())) {
-                    String message = format("Multiple group membership results for user [%s] with different distinguished names", user);
-                    log.debug(message);
-                    throw new AccessDeniedException(message);
+                    log.debug("Multiple group membership results for user [%s] with different distinguished names: [%s], [%s]", user, userDistinguishedName, search.next().getNameInNamespace());
```

Comment:
should you store `search.next().getNameInNamespace()` from `if`?

Reply:
Right, sloppy.